planetiler-custommap/README.md

```diff
@@ -675,6 +675,7 @@ docker run -v "$(pwd)/data":/data ghcr.io/onthegomap/planetiler:latest verify /d
   - `geometry` - Geometry type of the expected output feature.
   - `min_zoom` - Min zoom level that the output feature appears in.
   - `max_zoom` - Max zoom level that the output feature appears in.
+  - `min_pixel_size` - Minimum length of line features or square root of the minimum area of polygon features to emit below the maximum zoom-level of the map.
```

Comment:
Can we change the name to `min_size` and include in the description that the default unit is pixels? Eventually I'd like to extend it so you can specify `min_size` in units of pixels, tiles, meters, miles, acres, hectares, etc...

Reply:
Changed